src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1594,12 +1661,20 @@ impl application::View for GraphEditor {
           , (Press       , "" , "alt enter"         , "exit_node")
 
           // === Node Editing ===
-          , (Press   , "" , "cmd"                   , "edit_mode_on")
-          , (Release , "" , "cmd"                   , "edit_mode_off")
-          , (Press   , "" , "cmd enter"             , "edit_selected_node")
-          , (Press   , "" , "cmd left-mouse-button" , "edit_mode_on")
-          , (Release , "" , "cmd left-mouse-button" , "edit_mode_off")
-          , (Release , "" , "enter"                 , "stop_editing")
+          , (Press   , "!edge_creation_mode" , "cmd"                   , "edit_mode_on")
```

Comment:
I dont think this is good usage of kb masks. Instead, please handle it inside of the FRP. We will be removign kb masks in the future - it was bad idea to introduce them in the first place :(